src/man/include/ad_modified_defaults.xml

```diff
@@ -76,4 +76,21 @@
             </listitem>
         </itemizedlist>
     </refsect2>
+    <refsect2 id='nss_modifications'>
+        <title>NSS configuration</title>
+        <itemizedlist>
+            <listitem>
+                <para>
+                    fallback_homedir = /home/%d/%u
+                </para>
+		<para>
+		    If homeDirectory attribute is not present, the AD
+		    provider supplies its value automatically.
+		    If you have AD schema extended of posix attributes
```

Comment:
I think this sentence is not precise (or at least I don't understand it :-)).

First, the schema doesn't have to be extended with POSIX attributes, I think the schema is always there. It's just that with 2012 or earlier you can install a graphical UI add-on, but with 2016+ you need to go to the low-level attribute editor. So the question is not if you extend the schema, but if you use the POSIX attributes.

In general I wonder if it would be easiest to say something like "if you want to always rely on the values from AD LDAP, even if the value is missing, set fallback_homedir to %o"

Reply:
Please replace with:
```
The AD provider automatically sets "fallback_homedir = /home/%d/%u" to provide personal home directories for each use even if the homeDirectory attribute is missing. If your AD directory is properly populated with Posix attributes, and you want to avoid this fallback behavior, you can explicitly set "fallback_homedir = %o"